libraries/bxdf/mx39_open_pbr_surface.mtlx

```diff
@@ -0,0 +1,679 @@
+<?xml version="1.0"?>
+<materialx version="1.38">
+  <!--
+    OpenPBR Surface node definition backported to MaterialX 1.38.10
+    Snapshot taken at commit f98abc43 from June 16th 2024.
+  -->
+  <nodedef name="ND_open_pbr_surface_surfaceshader" node="open_pbr_surface" nodegroup="OpenPBR" version="1.1" isdefaultversion="true"
```

Comment:
Uses the official OpenPBR Surface node ID because this is the real deal, so no upgrade path necessary for this one.